metron-stellar/stellar-common/pom.xml

```diff
@@ -51,6 +52,11 @@
             </exclusions>
         </dependency>
         <dependency>
```

Comment:
Does there need to be a dependencies csv entry for this?

Reply:
Yep, added and dealt with merge conflicts.